index.html

```diff
@@ -1,7 +1,11 @@
 ---
 layout: default
 ---
-
+<!DOCTYPE HTML>
+<html>
```

Comment:
These should be inherited from `layout: default`. Are you not seeing them when the site is rendered?